src/main/java/uk/gov/hmcts/reform/pip/data/management/service/publication/PublicationManagementService.java

```diff
@@ -34,15 +35,15 @@ public class PublicationManagementService {
     private static final ObjectMapper MAPPER = new ObjectMapper();
 
     private final AzurePublicationBlobService azureBlobService;
-    private final ArtefactService artefactService;
+    private final PublicationRetrievalService artefactService;
     private final AccountManagementService accountManagementService;
     private final ListConversionFactory listConversionFactory;
     private final PublicationFileGenerationService publicationFileGenerationService;
     private final PublicationSummaryGenerationService publicationSummaryGenerationService;
 
     @Autowired
     public PublicationManagementService(AzurePublicationBlobService azureBlobService,
-                                        ArtefactService artefactService,
+                                        PublicationRetrievalService artefactService,
```

Comment:
rename artefactService to publicationRetrievalService

Reply:
changed to publicationRetrievalService